pages/web/hosting/htaccess_protect_directory_by_password/guide.fr-ca.md

```diff
@@ -64,8 +64,8 @@ Nous allons vous expliquer les 4 étapes principales à réaliser pour protéger
 > 
 > Pour modifier la version de PHP de vos sites web sur votre hébergement web, consultez les guides suivants :
 > 
-> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
-> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/php_configure_php_on_your_web_hosting_2014/)
+> - [Modifier la configuration de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime)
+> - [Changer la version de PHP de son hébergement web](/pages/web/hosting/ovhconfig_modify_system_runtime/)
```

Comment:
are you sure about this link ? (same as the one right above)

Reply:
/pages/web/hosting/php_configure_php_on_your_web_hosting_2014